Sources/XCTHealthKit/XCTest+CharacteristicEntry.swift

```diff
@@ -0,0 +1,206 @@
+            switch bloodType {
+            case .notSet:
+                picker.adjust(toPickerWheelValue: "")
+            case .aPositive:
+                picker.adjust(toPickerWheelValue: "A+")
+            case .aNegative:
+                picker.adjust(toPickerWheelValue: "A-")
+            case .bPositive:
+                picker.adjust(toPickerWheelValue: "B+")
+            case .bNegative:
+                picker.adjust(toPickerWheelValue: "B-")
+            case .abPositive:
+                picker.adjust(toPickerWheelValue: "AB+")
+            case .abNegative:
+                picker.adjust(toPickerWheelValue: "AB-")
+            case .oPositive:
+                picker.adjust(toPickerWheelValue: "O+")
+            case .oNegative:
+                picker.adjust(toPickerWheelValue: "O-")
+            @unknown default:
+                XCTFail("Unhandled blood type value: \(bloodType)")
+            }
+            healthApp.cells["Blood Type"].tap()
+        }
+        
+        if let skinType = characteristics.skinType {
+            healthApp.cells["Fitzpatrick Skin Type"].tap()
+            let picker = healthApp.pickers.firstMatch.pickerWheels.firstMatch
+            switch skinType {
+            case .notSet:
+                picker.adjust(toPickerWheelValue: "")
+            case .I:
+                picker.adjust(toPickerWheelValue: "Type I")
+            case .II:
+                picker.adjust(toPickerWheelValue: "Type II")
+            case .III:
+                picker.adjust(toPickerWheelValue: "Type III")
+            case .IV:
+                picker.adjust(toPickerWheelValue: "Type IV")
+            case .V:
+                picker.adjust(toPickerWheelValue: "Type V")
+            case .VI:
+                picker.adjust(toPickerWheelValue: "Type VI")
+            @unknown default:
+                XCTFail("Unhandled skin type value: \(skinType)")
+            }
+            healthApp.cells["Fitzpatrick Skin Type"].tap()
+        }
+        
+        if let wheelchairUse = characteristics.wheelchairUse {
+            healthApp.cells["Wheelchair"].tap()
+            let picker = healthApp.pickers.firstMatch.pickerWheels.firstMatch
+            switch wheelchairUse {
+            case .notSet:
+                picker.adjust(toPickerWheelValue: "")
+            case .no:
+                picker.adjust(toPickerWheelValue: "No")
+            case .yes:
+                picker.adjust(toPickerWheelValue: "Yes")
+            @unknown default:
+                XCTFail("Unhandled wheelchair use value: \(wheelchairUse)")
+            }
+            healthApp.cells["Wheelchair"].tap()
+        }
+        
+        healthApp.navigationBars["Health Details"].buttons["Done"].tap()
+        healthApp.navigationBars["Health Details"].buttons["Profile"].tap()
+        healthApp.navigationBars.firstMatch.buttons["Done"].tap()
+    }
+}
+
+
+extension XCUIElement {
+    func tryToTapReallySoftlyMaybeThisWillMakeItWork() {
```

Comment:
😂

Reply:
i added an explanatory comment to better document what's going on here